pyproject.toml

```diff
@@ -362,11 +360,12 @@ deps =
 setenv =
     COVERAGE_FILE = .coverage_{envname}
     DOC_PYTHON_VERSION = python3.11  # Keep this in sync with .readthedocs.yml and any CI scripts
-    # Skip pre-commit checks that are not needed (yamlfix should be removed from this list once Python 3.8 support is dropped)
-    SKIP = file-contents-sorter,yamlfix
+    # Skip pre-commit checks that are not needed
+    SKIP = file-contents-sorter
 commands_pre =
-    poetry install --no-root --without=main
+    poetry install --no-root --with tests,docs,dev
 commands =
+    !tests: python -c "import shutil; shutil.rmtree('.dist/{envname}', ignore_errors=True)"
```

Comment:
To use this, the build and twine steps below should use the same path as this line.

Reply:
And make sure to update the .gitignore file.

Reply:
The .gitignore file needs to be updated